xtensor-ioConfig.cmake.in

```diff
@@ -20,7 +20,6 @@ include(CMakeFindDependencyMacro)
 find_dependency(xtensor @xtensor_REQUIRED_VERSION@)
 
 set(PN xtensor_io)
-set_and_check(${PN}_INCLUDE_DIRS "${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_INCLUDEDIR@")
 set(${PN}_LIBRARY "")
 check_required_components(${PN})
```

Comment:
I think it's fine to remove these 4 lines (22 to 25) like you did previously. We removed them in other packages of the xtensor stack.